src/src/main/java/com/hcmus/mentor/backend/infrastructure/fileupload/BlobStorage.java

```diff
@@ -16,5 +24,6 @@ public interface BlobStorage {
 
   Boolean exists(String key);
 
-  String getUrl(String key);
+  String getUrl(String key)
+      throws ServerException, InsufficientDataException, ErrorResponseException, IOException, NoSuchAlgorithmException, InvalidKeyException, InvalidResponseException, XmlParserException, InternalException;
```

Comment:
Separate this line into multiple lines

Reply:
Split this line into multiple lines but the code formatter has merged them. Should me separate and not reformat this file